src/utils/update/trust.model.fetch.ts

```diff
@@ -87,7 +87,7 @@ export const mapTrustData = (trustData: TrustData, appData: ApplicationData) =>
     creation_date_month: dateOfBirth?.month ?? "",
     creation_date_year: dateOfBirth?.year ?? "",
     trust_still_involved_in_overseas_entity: stillInvolved,
-    unable_to_obtain_all_trust_info: trustData.unableToObtainAllTrustInfoIndicator ? "Yes" : "No",
+    unable_to_obtain_all_trust_info: trustData.unableToObtainAllTrustInfoIndicator ? "No" : "Yes",
```

Comment:
This still seems to be happening, instead of always displaying No , it now always displays Yes. Even when you change the value in the database from N to Y and Y to N it always shows Yes

Reply:
review addressed